terraform/ec2/win/install_and_start_agent.tpl

```diff
@@ -0,0 +1,22 @@
+<powershell>
+$installDirectory = "c:\temp\cw"
+$downloadDirectory = $installDirectory
+$logsDirectory = $installDirectory
+$cwAgentInstaller = "$downloadDirectory\amazon-cloudwatch-agent.msi"
+$cwAgentInstallPath = "C:\\Program Files\\Amazon\\AmazonCloudWatchAgent"
+
+New-Item -ItemType "directory" -Path $installDirectory
+
+Set-Location -Path $installDirectory
+
+Write-host "Installing Cloudwatch Agent"
+Install-PackageProvider NuGet -Force;
+Set-PSRepository PSGallery -InstallationPolicy Trusted
+Install-Module -Name AWS.Tools.EC2,AWS.Tools.S3 -AllowClobber
```

Comment:
Can you add a comment as to why we need these tools installed?

Reply:
This is for the aws cli commands. Powershell uses this instead of the normal cli.

Reply:
Added